DataFormats/simulation/include/SimulationDataFormat/MCCompLabel.h

```diff
@@ -21,21 +21,24 @@ class MCCompLabel
 {
  private:
   static constexpr ULong64_t ul0x1 = 0x1;
+  static constexpr ULong64_t ALLBits = 0xffffffffffffffff;
   static constexpr ULong64_t NotSet = 0xffffffffffffffff;
+  static constexpr ULong64_t Invalid = ul0x1 << 63;
```

Comment:
Not sure if we need a full bit for invalid flags. It should be enough to have one single invalid value like 0xfffffffffffffffe, or in case you want to keep the event and source id, we can use 0x7fffffff as track ID.
In this way we save one bit, which could later be used for the weight.

Reply:
ok